netcore/Makefile

```diff
@@ -183,14 +183,25 @@ corefx/.stamp-dl-corefx-tests-$(NETCORETESTS_VERSION):
 #
 
 run-tests-corefx: update-tests-corefx
+	@rm -f .failures; \
+	counter=0; \
+	tests_count=$(words $(dir $(wildcard corefx/tests/extracted/*/))); \
 	for testdir in corefx/tests/extracted/*; do \
-		$(MAKE) run-tests-corefx-$$(basename $${testdir}) || true; \
+		counter=$$((counter+1)); \
+		$(MAKE) run-tests-corefx-$$(basename $${testdir}) TEST_COUNTER="($$counter / $$tests_count) " || echo $$(basename $$testdir) >> .failures; \
```

Comment:
the CI step used to run `run-step.sh` to execute each set in order to cancel if timeout is exceeded.

Reply:
Thanks, I added the timeout step back (using the `timeout` command directly to keep a separation between the Jenkins stuff and netcore).